src/channels/receiver.rs

```diff
@@ -253,6 +255,38 @@ where
         })
     }
 
+    ///
+    /// Sends message over chromecast message bus
+    ///
+    /// Receiver can observe messages using `context.addCustomMessageListener` with custom namespace.
+    ///
+    ///```javascript, no_run
+    /// context.addCustomMessageListener('urn:x-cast:com.example.castdata', function(customEvent) {
+    /// // do something with message
+    /// });
+    ///```
+    ///
+    /// Namespace should start with `urn:x-cast:`
+    ///
+
+    pub fn send_message<M: Serialize>(&self, namespace: &str, message: &M) -> Result<(), Error> {
```

Comment:
__question:__ since we set destination to `*` would it make sense to name this method (and reflect in rust docs) as `broadcast_message` instead (no specific destination == broadcast to any interested party)?

Reply:
I have no strong opinion about this. I was looking into exposing destination as a parameter but do not see there can be two different receivers running at the same time

Reply:
Android API uses send 

```
Cast.CastApi.sendMessage(mApiClient,
                    mHelloWorldChannel.getNamespace(), message)

```

Reply:
Interesting, I'm wondering if they just use specific `destination/receiver` under the hood. Having said that, did you pick `*` as a destination because `destination: self.receiver.to_string()` didn't work for you for some reason? If `self.receiver` works then `send_message` would also make sense (sorry, just slowly recovering in memory how that code works, haven't looked at it in ages).

Reply:
`self.receiver.to_string()` did not work, I've spent some time trying to locate is it issue wits sender, receiver or library, then i realise debug messages are arriving with `destination: *`

Reply:
Okay, let's keep it as is then.

Reply:
I’ve renamed it to broadcast, do you want me to revert it back to send?

Reply:
Nope, I meant let's keep broadcast for now, everything is good :+1: